src/pages/List/components/DonationList/DonationElement/index.jsx

```diff
@@ -0,0 +1,42 @@
+import styles from './styles.module.scss';
+import donationWonpil from '../../../../../assets/images/donation/day6_wonpil.jpg';
+import creditImg from '../../../../../assets/images/img_diamond.png';
```

Comment:
자기 컴포넌트 디렉토리를 제외한 외부에서 가져오는 것들은 절대경로로 쓰면 좋습니다~
```suggestion
import donationWonpil from '@/assets/images/donation/day6_wonpil.jpg';
import creditImg from '@/assets/images/img_diamond.png';
```

Reply:
아! 절대/ 상대 경로 구분 기준일 살짝 헷갈렸었는데 상기시켜주셔서 감사합니다 ㅎㅎ